src/getCoinCombination.test.js

```diff
@@ -4,7 +4,30 @@ describe('getCoinCombination', () => {
   const { getCoinCombination } = require('./getCoinCombination');
 
   it(`should be declared`, () => {
-    expect(getCoinCombination)
-      .toBeInstanceOf(Function);
+    expect(getCoinCombination).toBeInstanceOf(Function);
+  });
+
+  it('should return [1, 0, 0, 0] when input value is (1)', () => {
```

Comment:
'input value is (1)' looks weird and wrong as for me. better to write: input value = 1 or 'coins' = 1

Reply:
Fixed